src/wp-includes/post.php

```diff
@@ -7805,7 +7805,7 @@ function _prime_post_caches( $ids, $update_term_cache = true, $update_meta_cache
  *
  * @param int[] $ids ID list.
  */
-function _prime_post_parents_caches( array $ids ) {
+function _prime_post_parent_id_caches( array $ids ) {
```

Comment:
Can we bring over the other docblock improvements from @peterwilsoncc's PR ([here](https://github.com/WordPress/wordpress-develop/pull/5386/files#diff-ef9c5475561d459917dac57d21e08a6dfeb36f8b1c44b978d3696aec8b97c405R7811-R7814)) as well?

Reply:
@joemcgill Done.